### PR TITLE
doc: fix missed doc and help msg about firewall plugin

### PR DESCRIPTION
**Pull request by t1anz0ng**
Signed-off-by: tianyang ni <tianzong48@gmail.com>

### CODE REVIEW

**Comment by fahedouch** [comment]
We should mention that `isolation plugin` is deprecated

**Comment by AkihiroSuda** [comment]
Thanks

File: README.md [hunk 1]
```diff
@@ -978,7 +978,7 @@ Usage: `nerdctl logout [SERVER]`
 ### :whale: nerdctl network create
 Create a network
 
-:information_source: To isolate CNI bridge, [CNI isolation plugin](https://github.com/AkihiroSuda/cni-isolation) needs to be installed.
+:information_source: To isolate CNI bridge, CNI plugin [firewall](https://www.cni.dev/plugins/current/meta/firewall/)(>= `v1.1.0`), or [CNI isolation plugin](https://github.com/AkihiroSuda/cni-isolation) needs to be installed.
```
Comment (AkihiroSuda) [inline README.md:981]:
```suggestion
:information_source: To isolate CNI bridge, CNI plugins v1.1.0 or later needs to be installed.
```

Probably no need to mention the deprecated isolation plugin

Comment (t1anz0ng) [reply README.md:981]: SGTM

**Review by AkihiroSuda — APPROVED** [review]

**Comment by t1anz0ng** [comment]
@fahedouch PTAL, thanks!

**Pull request merged**